_content/vulnerability_disclosure.yml

```diff
@@ -0,0 +1,65 @@
+       - Only use exploits to the extent necessary to confirm a vulnerability’s presence. Do not use an exploit to compromise or exfiltrate data, establish persistent command line access, or use the exploit to pivot to other systems.
+       - Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.
+       - Do not submit a high volume of low-quality reports.
+
+    Once you’ve established that a vulnerability exists or encounter any sensitive data (including personally identifiable information, financial information, or proprietary information or trade secrets of any party), you must stop your test, notify us immediately, and not disclose this data to anyone else.
+
+test_methods:
+  title: 'Test methods'
+  section_text: |-
+      The following test methods are not authorized:
+
+      - Network denial of service (DoS or DDoS) tests or other tests that impair access to or damage a system or data
+
+      - Physical testing (e.g. office access, open doors, tailgating)
+
+      - Social engineering (e.g. phishing, vishing), or any other non-technical vulnerability testing
+
+scope:
+  title: 'Scope'
+  section_text: |-
+    This policy applies to the following systems and services:
+
+    - *.usds.gov
+    
+    - *.unitedstatesdigitalservice.gov
+ 
+    Any service not expressly listed above, such as any connected services, are excluded from scope and are not authorized for testing. Additionally, vulnerabilities found in systems from our vendors fall outside of this policy’s scope and should be reported directly to the vendor according to their disclosure policy (if any). If you aren’t sure whether a system is in scope or not, contact us at <infrastructure@usds.gov> before starting your research.
+ 
+    Though we develop and maintain other internet-accessible systems or services, we ask that active research and testing only be conducted on the systems and services covered by the scope of this document. If there is a particular system not in scope that you think merits testing, please contact us to discuss it first. We will increase the scope of this policy over time.
+
+reporting:
+  title: 'Reporting a vulnerability'
+  section_text: |-
+    Information submitted under this policy will be used for defensive purposes only – to mitigate or remediate vulnerabilities. If your findings include newly discovered vulnerabilities that affect all users of a product or service and not solely United States Digital Service, we may share your report with the other organizations as needed to coordinate and remediate the vulnerability. We will not share your name or contact information without express permission.
+
+    We accept vulnerability reports via <infrastructure@usds.gov>. Reports may be submitted anonymously. If you share contact information, we will acknowledge receipt of your report within 3 business days.
```

Comment:
Do we want to address if we do/do not support PGP e-mails?

Reply:
addressed in 4a279f9. 
screencap of affected section: 

<img src='https://user-images.githubusercontent.com/31673962/109004594-f2ffc900-7676-11eb-8872-bfd33e18f31d.png' width=200/>